scripts/util/dont-show-again.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const PS_OJBECT_KEY = 'dosa';
+
+/**
+ * A utility of handling anything with DOn't Show Again behaviour (not rice-based pancakes).
+ */
+class DosaHandler {
```

Comment:
Really dumb nit probably but since DOSA is an acronym (sorta) it should probably be `DOSAHandler` here.

Reply:
Nevermind, virgin C++ naming convention versus JS gigachad naming convention